test/common/testlib.py

```diff
@@ -502,7 +503,62 @@ def mouse(
         :param metaKey: press the meta key
         """
         self.wait_visible(selector)
-        self.call_js_func('ph_mouse', selector, event, x, y, btn, ctrlKey, shiftKey, altKey, metaKey)
+
+        # HACK: Chromium clicks don't work with iframes; use our old "synthesize MouseEvent" approach
+        # https://issues.chromium.org/issues/359616812
+        # TODO: x and y are not currently implemented: webdriver (0, 0) is the element's center, not top left corner
+        if self.browser == "chromium" or x != 0 or y != 0:
```

Comment:
I kinda lean to getting rid of x/y as API:

```
test/verify/check-storage-lvm2:            b.mouse(slider, "click", width, 0)
test/verify/check-storage-partitions:        b.mouse(slider, "click", about_half_way, 0)
```

So this usually relates to the thing you want to click. So once the chromium bug is resolved I'd be for changing this to the BiDi API as well and making it the x/y coordinate relative to the clicked element (this feels very logical).

Reply:
The x/y coords *are* relative to the clicked element of course -- but for [MouseEvent](https://developer.mozilla.org/en-US/docs/Web/API/MouseEvent)  (i.e. via `ph_mouse()` (0, 0) is the top left, while with webdriver (0, 0) is the element's center.

Once we can run chromium with BiDi as well, then indeed I'm all for just redefining x/y here, and adjusting tests to specify the delta relative to the center. That's just impractical with this "half pm_mouse/half BiDi" state.

Reply:
Then we are on the same page :+1: